src/main/java/screen/GameScreen.java

```diff
@@ -288,8 +293,9 @@ public final void initialize() {
 		this.screenFinishedCooldown = Core.getCooldown(SCREEN_CHANGE_INTERVAL);
 		this.bullets = new HashSet<>();
 		this.barriers = new HashSet<>();
+		this.allyShips = new HashSet<>();
         this.itemBoxes = new HashSet<>();
-		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation, this.barriers, this.width, this.height, this.balance);
+		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation, this.barriers, this.allyShips, this.height, this.width, this.balance);
```

Comment:
The line is not resolved yet. The order of `this.width` and `this.height` should be swapped.

```suggestion
		this.itemManager = new ItemManager(this.ship, this.enemyShipFormation, this.barriers, this.allyShips, this.width, this.height, this.balance);
```